### DISCUSSION TITLE
Trikz plugin

### DISCUSSION THREAD

**Discussion by Sunnymittal112** [community/community#121322] [Other Feature Feedback, Questions, & Ideas]

**Comment by ebndev** [comment]
Hi @Sunnymittal112,

Thanks for being a part of the GitHub Community, we're glad you're here!

It looks like you're looking for support from CS Source. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
